networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/libp2p/LibP2PNetwork.java

```diff
@@ -116,6 +117,7 @@ public LibP2PNetwork(
             b -> {
               b.getIdentity().setFactory(() -> privKey);
               b.getTransports().add(TcpTransport::new);
+              b.getSecureChannels().add(NoiseXXSecureChannel::new);
```

Comment:
By adding Noise above Secio, do we preferentially use Noise now?

Reply:
Yes. When initiating connection we would try to negotiate `noise` first